app/services/FileTransferService.scala

```diff
@@ -77,27 +83,32 @@ class FileTransferService @Inject()(actorSystem: ActorSystem,
     }
 
     actorSystem.scheduler.scheduleOnce(0 milliseconds) {
-      val allResponses = requests.foldLeft(Future(List.empty[FileTransferResponse])) {
+      val allResponses: Future[List[FileTransferResponse]] = requests.foldLeft(Future(List.empty[FileTransferResponse])) {
         (previousResponses, req) ⇒
           for {
             responses ← previousResponses
             response ← connector.transferFile(req)(hc, implicitly)
           } yield responses :+ response
       }
-
-      allResponses.flatMap(auditFileTransfers)
+      allResponses.map(file => auditFileTransfers(file, caseId))
+        .onComplete{
```

Comment:
Is this not already logged in the auditFileTransfers functions? I would rather not wait for this to complete before returning back to the calling function.